### ISSUE TITLE
Barrier prevents screens from going to sleep

### ISSUE THREAD

**Issue by mirh** [debauchee/barrier#436]
### Operating Systems ###

Server: Windows 7

Client: Manjaro Linux

### Barrier Version ###

2.31 git/snapshot

### Steps to reproduce bug ###

1. Connect the two computers
2. Don't touch them for the amount of time that it should take for the monitors to turn off
3. They always keep on

**Comment by cowbe0x004** [comment]
Check out this thread
https://members.symless.com/forums/topic/4794-does-synergy-prevent-monitor-from-going-to-sleep/

**Comment by mirh** [comment]
I'm sure that'll work
But still, I don't understand whatever rationale was behind [this](https://github.com/debauchee/barrier/blob/v2.3.1/src/lib/platform/XWindowsScreen.cpp#L266) [code](
https://github.com/debauchee/barrier/blob/v2.3.1/src/lib/platform/MSWindowsScreen.cpp#L1450) to begin with. 
And why nobody still didn't fix it.

**Comment by mirh** [comment]
Ok, after sleeping on it, I guess like it may even have its sense to have this kind of feature 
(assuming "virtual" inputs aren't treated the same of physical ones power-management-wise... which actually seemed to be the case after reading some code, but I could easily be wrong)
You wouldn't like your client to go to sleep while you are remote-ing in it. 

Why the hell is the same thing also being enforced on the host though? And would it have taken too much time to add an activity/screen check?

**Comment by remotejob** [comment]
The same problem
in my case:

Operating Systems
Server: Ubuntu 18.04
Client: Ubuntu 18.04

Barrier Version
commit 0ed94514306f89aed586a2d90505bbd0f67d78ea 
Date:   Wed Aug 28 13:18:48 2019 -0400

But only Client ON not to go to sleep.

**Comment by OptimusGREEN** [comment]
yeah im currently struggling with this on an ubuntu 22.04 client. It would be great if there was a solution soon.

**Comment by andreysm** [comment]
Encountered this on Windows.
Disabling "Synchronize screen savers" checkbox at "Advanced server settings" tab solved the problem.
The system doesn't go to sleep, but the screen does. (`powercfg /requests` is informative.)

**Comment by vathanac** [comment]
> Encountered this on Windows. Disabling "Synchronize screen savers" checkbox at "Advanced server settings" tab solved the problem. The system doesn't go to sleep, but the screen does. (`powercfg /requests` is informative.)

I can't find where to disable "Synchronize screen savers". Can you elaborate?

**Comment by andreysm** [comment]
![barrier-settings](https://github.com/user-attachments/assets/b413b06e-2355-484c-a314-a5861ced3a73)
